hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
-* Coordinate with one of the hosters providing free mumble servers and start promoting Mumble as the voicechat of choice for your favorite game.
-* Add mumble hosters you found to the {{< wiki Hosters />}} wiki page / update information.
-* Create How-Tos, Screencasts, Tutorials or Benchmarks on typical issues and topics, and publish them on prominent platforms.
-* Spread the word in groups – join our [Steam Group](https://steamcommunity.com/groups/Mumble_VoIP)
+<!-- Add more details, descriptions etc.? -->
+You find the Mumble theme in it's own [repository on Github](https://github.com/mumble-voip/mumble-theme).
 
-# Skinning
+#### Skins
 
 Mumble skins bring variety into Mumbles design and choices for different tastes.
 
 Create new skins for Mumble! Information can be found on our {{< wiki Skinning "Skinning wiki page" />}} and existing skins we know about on the {{< wiki Skins "Skins wiki page" />}}.
 
-# Coding
 
-## 3rd party Interfaces
+# Documentation & Website
 
-Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based RPC Interface (using the deprecated {{< wiki DBus />}} Interface is no longer advised). You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with Murmur might look like you can take a look at the example scripts in our git repositories script folder.
+Improve our Documentation and Website.
 
-## Ice authenticators
+## Documentation
 
-See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement you and the community can bring us!
+<!-- Todo: 
 
-## Positional Audio
+Add a concrete plan:
+- what documentation we want to have
+- what should be converted from the wiki
+- what is done & what is not
 
-Most of our positional audio plugins regularly break on game updates. To read on how you can help us update the plugins see the {{< wiki Pluginguide />}} page. You can even fix the update-issue forever for open source games, or if you want to add positional audio to your game, see {{< wiki Link />}}.
+- add issues and open pull requests
 
-## Mumble itself
+-->
+
+We plan to move all our documentation to the Website and Github Repo respectively.
+
+<!-- 
+
+**Todo:**
+
+- Convert parts of the Wiki into Markdown files
+- Rewrite parts of the documentation, so it is more recent, comprehensive and fits the style <!-- of the website, markdown files on github -->
+
+Right now you find most of Documentation in our
+[wiki](https://wiki.mumble.info).
+Any contributions are welcome, improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+
+The new places for documentation will be here:
+
+- [Mumble Github Repo - Docs](https://github.com/mumble-voip/mumble/tree/master/docs)
```

Comment:
```suggestion
- [Mumble GitHub repo - Docs](https://github.com/mumble-voip/mumble/tree/master/docs) for developer-related documentation
```

Reply:
Regarding my pull request for user-guides that is undecided for now.
But if you decide it should be solely on the website...

Reply:
This is actually something that we have not (yet) decided as a team yet. That's solely what I think would make the most sense.
Maybe @davidebeatrici and @Kissaki can given their opinion on this as well?

Regarding your pull request: Given that these are Markdown files anyways I don't think it matters all that much as to in which repo it gets initially created as it can be easily migrated if needed.